### PR TITLE
Fixes onevnet type & specs.

### PR DESCRIPTION
**Pull request by sebastian-su**

### CODE REVIEW

File: spec/acceptance/onevnet_spec.rb [hunk 1]
```diff
@@ -10,144 +10,112 @@ class { 'one':
...
-          bridge       => 'vbr0',
-          macstart     => '02:00:c0:a8:00:01',
-          network_size => 'C',
-          siteprefix   => 'fd12:33a:df34:1a::',
-          globalprefix => '2004:a128::',
-        }
+      onevnet_addressrange { 'ar2':
+          ensure        => present,
+          onevnet_name  => 'vnet2',
+          ar_id         => '2',
```
Comment (tuxmea) [inline spec/acceptance/onevnet_spec.rb]: ar_id is a read only value. you should not set that attribute in type declaration

File: spec/acceptance/onevnet_spec.rb [hunk 2]
```diff
@@ -10,144 +10,112 @@ class { 'one':
...
-          bridge          => 'vbr0',
-          network_address => '192.168.1.0',
-          network_size    => 'C',
-          gateway         => '192.168.1.1',
-          dnsservers      => '192.168.1.1',
-        }
+      onevnet_addressrange { 'ar3':
+          ensure        => present,
+          onevnet_name  => 'vnet2',
+          ar_id         => '2',
```
Comment (tuxmea) [inline spec/acceptance/onevnet_spec.rb]: ar_id is a read only value. you should not set that attribute in type declaration

**Comment by tuxmea** [comment]
looks good to me - except for the ar_id usage on the onevnet_addressrange type.

File: spec/acceptance/onevnet_spec.rb (hunk 2 above)
Comment (sebastian-su) [inline spec/acceptance/onevnet_spec.rb]: ah ok, will fix it - need to fix the tests anyways

**Pull request merged**